### ISSUE TITLE
Accessibility: Embedded youtube video should show auto-generated captions

### ISSUE THREAD

**Issue by Yavnikaa** [IMGIITRoorkee/chakra-library#215]
YouTube iframe embeds lack `cc_load_policy=1` parameter to enable auto-generated captions. Affects accessibility for hearing-impaired users.
